tests/cases/conformance/jsdoc/checkJsdocOnEndOfFile.ts

```diff
@@ -0,0 +1,8 @@
+// @noEmit: true
```

Comment:
Can you instead use outFile to verify emit.

Reply:
Sure, although, there's no change from the input JS. Is there reason to expect that a jsdoc on the EOF token would get dropped?